src/adagrams.js

```diff
@@ -1,15 +1,143 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
```

Comment:
As we build this program out, we might use letterPool for other functions, so it might be better implemented as a global constant. Just remember that if we do that, we make global constants upper snakeCase so it would be LETTER_POOL instead.